Cabal/src/Distribution/Utils/IOData.hs

```diff
@@ -53,7 +54,9 @@ class NFData mode => KnownIODataMode mode where
   toIOData :: mode -> IOData
   iodataMode :: IODataMode mode
 
--- | @since 3.2
+-- | Phantom-typed GADT representation of IO data mode,
+-- containing no other data.
+-- @since 3.2
```

Comment:
Have you tried running `cabal haddock` on this? Is the `since xyz` string displayed correctly (i.e. in *italics*)?